views/events/calls_events.view.lkml

```diff
@@ -205,22 +210,118 @@ view: calls_events {
     sql: ${TABLE}."UUID_TS" ;;
   }
 
+  dimension_group: active_user_date {
+    label: "Active User"
+    description: "Use with Active User/Instance Dimensions to enable Daily, Weekly & Monthly active user/instance functionality with this explore."
```

Comment:
Does this work for anything, or just for Calls?

Reply:
It works for anything, its leverages the dates table join.